lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMBase.h

```diff
@@ -200,9 +202,10 @@ class ConvertTritonGPUOpToLLVMPatternBase {
 
   explicit ConvertTritonGPUOpToLLVMPatternBase(
       TritonGPUToLLVMTypeConverter &typeConverter, ModuleAllocation &allocation,
-      IndexCacheInfo indexCacheInfo)
+      IndexCacheInfo indexCacheInfo,
+      std::map<std::string, int> *pStat = nullptr)
```

Comment:
Same, does the code actually handle null pStat?  (And is the complexity of making the code handle it worthwhile?)